webauthn-server-core/src/main/java/com/yubico/webauthn/RegistrationResult.java

```diff
@@ -30,17 +30,7 @@
 import com.yubico.internal.util.CertificateParser;
 import com.yubico.webauthn.RelyingParty.RelyingPartyBuilder;
 import com.yubico.webauthn.attestation.AttestationTrustSource;
-import com.yubico.webauthn.data.AttestationType;
-import com.yubico.webauthn.data.AuthenticatorAttachment;
-import com.yubico.webauthn.data.AuthenticatorAttestationResponse;
-import com.yubico.webauthn.data.AuthenticatorData;
-import com.yubico.webauthn.data.AuthenticatorDataFlags;
-import com.yubico.webauthn.data.AuthenticatorRegistrationExtensionOutputs;
-import com.yubico.webauthn.data.AuthenticatorResponse;
-import com.yubico.webauthn.data.ByteArray;
-import com.yubico.webauthn.data.ClientRegistrationExtensionOutputs;
-import com.yubico.webauthn.data.PublicKeyCredential;
-import com.yubico.webauthn.data.PublicKeyCredentialDescriptor;
+import com.yubico.webauthn.data.*;
```

Comment:
Please configure your IDE to use single imports rather than wildcard ones. I'll fix this and add a note to the developer docs referencing the [Google style guide](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).

Reply:
dev-docs PR: https://github.com/Yubico/java-webauthn-server/pull/367